### DISCUSSION TITLE
Inverse transform values

### DISCUSSION THREAD

**Discussion by insuquot** [MAIF/shapash#147] [Q&A]
I’ve trained a model with scaled data, how can I use tell shapash to display the unscaled values?

**Comment by yg79** [comment]
If you use standard scaler of sklearn ColumnTransformers, you can specify it in the compile step to display the unscaled values.
more detail in this tutorial:
https://github.com/MAIF/shapash/blob/master/tutorial/encoder/tuto-encoder02-using-columntransformer.ipynb
